spec/interactors/create_password_reset_token_spec.rb

```diff
@@ -0,0 +1,35 @@
+require "rails_helper"
+
+RSpec.describe CreatePasswordResetToken do
+  describe ".call" do
+    let(:user) { create(:confirmed_user, reset_token: nil) }
+
+    context "when successful" do
+      subject do
+        described_class.call(user: user)
+      end
+
+      it "sets the user's reset token" do
+        expect(subject.user.reset_token).to eq(subject.raw)
+      end
+
+      it "sets the user's reset_digest" do
+        expect(subject.user.reset_digest).to eq(subject.enc)
+      end
+
+      it "sets the user's reset_sent_at" do
+        expect(subject.user.reset_sent_at).to be_an(ActiveSupport::TimeWithZone)
```

Comment:
I am not checking that these attributes are actually set correctly to `nil` before this interactor is called. I was thinking I should probably do this when I get to the interactor that creates the User? I.e. check that `reset_token`, `reset_digest`, etc. are all initialized to `nil`? Or should I be checking that stuff here, prior to calling `CreatePasswordResetToken`?